core/src/main/java/com/amplifyframework/core/DevMenuLogsFragment.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.Fragment;
+
+/**
+ * A {@link Fragment} subclass representing the view
+ * to display the logs on the developer menu.
+ */
+public final class DevMenuLogsFragment extends Fragment {
```

Comment:
This template is repeated a few times.

Can you make just one of these fragments, and give it a constructor which accepts:
```
public final class DevMenuLogsFragement extends Fragment {
    @IdRes private final int layoutResourceId;

    public DevMenuLogsFragment(@IdRes int layoutResourceId) {
        this.layoutResourceId = layoutResourceId;
    }
    ...
}
```

___Update___: oh, duh. Nope. Your comment says the no-args constructor is required. Nevermind!